src/pytest/test.py

```diff
@@ -960,24 +950,20 @@ def testGeo(env):
         env.assertEqual(len(hotels), res[0])
 
         res = gsearch('hilton', "-0.1757", "51.5156", '1')
-        print res
         env.assertEqual(3, res[0])
-        env.assertEqual('hotel2', res[5])
-        env.assertEqual('hotel21', res[3])
-        env.assertEqual('hotel79', res[1])
+        env.assertIn('hotel2', res)
+        env.assertIn('hotel21', res)
+        env.assertIn('hotel79', res)
         res2 = gsearch_inline('hilton', "-0.1757", "51.5156", '1')
         env.assertListEqual(res, res2)
 
         res = gsearch('hilton', "-0.1757", "51.5156", '10')
         env.assertEqual(14, res[0])
-        env.assertEqual('hotel93', res[1])
-        env.assertEqual('hotel92', res[3])
-        env.assertEqual('hotel79', res[5])
```

Comment:
?

Reply:
order changes between reloads. We have the count